tests/test_api.py

```diff
@@ -30,7 +31,8 @@ def setUp(self):
         self.api = API(self.server, self.username, self.password, str(self.totp.now()))
         for i in range(3):
             try:
-                self.api.authenticate()
+                result = self.api.authenticate()
+                self.assertTrue(result is True)
```

Comment:
We're using `pytest` so we don't need the `self.assertXXX` from `unittest` to get nice error messages.

Reply:
Here I was just following the existing tests.

Reply:
Yup, as the initial tests were for `unittest` module :)